### ISSUE TITLE
Current folder colors incorrect until Refresh visible tabs command

### ISSUE THREAD

**Issue by rakoczy** [RealRaven2000/QuickFolders#166]
After TB starts current folder is not marked with color until Refresh visible tabs command is issued (from  Quickfolders Options). After that everything is OK until the next TB start.

TB 78.11.0 (64-bit)
QF Version 5.3 Last Updated March 17, 2021

**Comment by RealRaven2000** [comment]
> After TB starts current folder is not marked with color until Refresh visible tabs command is issued (from Quickfolders Options). After that everything is OK until the next TB start.
> 
> TB 78.11.0 (64-bit)
> QF Version 5.3 Last Updated March 17, 2021

Is there a reason why you're not using the latest version of QuickFolders? The latest version is 5.5.2 ...
if you have automatic updates for Add-ons enabled, you may be experiencing issue 165 which requires you to click on the menu button for updating permissions, does it look like this?

![image](https://user-images.githubusercontent.com/6314674/122197450-181b8000-ce90-11eb-9aa6-5c25446de1bd.png)

**Comment by RealRaven2000** [comment]
> After TB starts current folder is not marked with color until Refresh visible tabs command is issued (from Quickfolders Options). After that everything is OK until the next TB start.
> 
> TB 78.11.0 (64-bit)
> QF Version 5.3 Last Updated March 17, 2021

What theme are you using?? I did some tests with non-standard themes and there seems to be a system variable `--lwt-toolbar-field-color` not set for 3rd party themes. It appears to work ok for the build in themes (default, dark, and Light). Note that the colors are not instantly refreshed for the toolbar when you _change to a different type of theme with TB running_, so you still have to refresh folders in this case to test. 

try this test version (to install, download the zip file and drag into Thunderbird Add-ons Manager)
[QuickFolders-mx-5.7pre13.zip](https://github.com/RealRaven2000/QuickFolders/files/6830255/QuickFolders-mx-5.7pre13.zip)

I have changed the container rule as follows

```
.QuickFolders-NavigationPanel {
-  color: var(--lwt-toolbar-field-color,black); 
+  color: var(--lwt-text-color, --lwt-toolbar-field-color); 
}
```

**Comment by RealRaven2000** [comment]
Latest version, which adds more fixes to the Current Folders colors making it more theme-compatible:

[QuickFolders-mx-5.7pre17.zip](https://github.com/RealRaven2000/QuickFolders/files/6836091/QuickFolders-mx-5.7pre17.zip)

(to install, download the zip file and drag into Thunderbird Add-ons Manager)

**Comment by RealRaven2000** [comment]
Fixed in 5.7, [released on ATN](https://addons.thunderbird.net/thunderbird/addon/quickfolders-tabbed-folders/versions/5.7) today.
